_sessions/alex-henman.md

```diff
@@ -7,7 +7,7 @@ permalink: /sessions/alex-henman/
 name_slug: alex-henman
 session_type: Talk
 session_title: "Beyond faceted search"
-photo_url: https://www.gravatar.com/avatar/750750a74bce18e3bf0831510b458049
+photo_url: https://pretalx.com/media/avatars/IMG-20221018-WA00031_vIjyR0B.jpg
```

Comment:
@rebkwok you can delete the image of alex if it's already generated and re-launch the command to re-generate it I think